src/org/intellij/markdown/html/HtmlGenerator.kt

```diff
@@ -27,12 +28,13 @@ class HtmlGenerator(private val markdownText: String,
     
     private val htmlString: StringBuilder = StringBuilder()
 
-    fun generateHtml(): String {
-        HtmlGeneratingVisitor().visitNode(root)
+    @JvmOverloads
+    fun generateHtml(customizer: AttributesCustomizer? = null): String {
```

Comment:
Do I understand correctly that `@JvmOverloads` will preserve ABI compatibility in that case?

Reply:
Yes, it will, by providing an overloaded method.